HighLevelGoals.md

```diff
@@ -17,8 +17,7 @@
       other than C/C++.
 3. Design to execute within and integrate well with the *existing*
    [Web platform](Web.md):
-    * maintain the versionless, [feature-tested](FeatureTest.md) and
```

Comment:
Can you add a TODO comment here, so that someone may reintroduce this section later?